src/fides/api/schemas/connection_configuration/saas_config_template_values.py

```diff
@@ -17,3 +18,18 @@ class SaasConnectionTemplateValues(BaseModel):
 
     class Config:
         extra = Extra.ignore
+
+    def generate_config_data_from_template(
```

Comment:
i think pydantic objects must be proper instances that can be accessed this way, right? tbh i'm actually not 100% sure and pydantic can feel a bit 🪄  sometimes, so this is the main assumption that needs to be validated!

Reply:
from my initial testing, i think this does work as expected 👍